### ISSUE TITLE
Plugin doesn't appear to start

### ISSUE THREAD

**Issue by bdashrad** [sftpgo/sftpgo-plugin-pubsub#1]
The sftpgo-plugin-pubsub never appears to start when defined in the plugins section of the config. I see no log messages for it, even when I give it a correct or an incorrect configuration, which both trigger log messages when running the plugin independently. 

Here is the excerpt of our config for the plugins. The plugin is built using `go build` and then marked as executable. 

sftpgo v2.1.0, problem observed in the osx version as well as the alpine linux container `ghcr.io/drakkan/sftpgo:v2.1.0`

Am I missing something in the config definition?

```yaml
common:
  proxy_protocol: 1
data_provider:
  driver: postgresql
  name: sftpgo
  host: "127.0.0.1"
  port: 5432
  username: sftpgo
  prefer_database_credentials: true
plugins:
  - type: notifier
    notifier_options:
      fs_events:
        - upload
        - download
      user_events:
        - add
        - update
      retry_max_time: 60
      retry_queue_max_size: 1000
    cmd: /plugins/sftpgo-plugin-pubsub/sftpgo-plugin-pubsub
    args:
      - "gcppubsub://projects/my-project/topics/my-topic"
    auto_mtls: true
```

**Comment by drakkan** [comment]
Hi,

to test plugins you have to use the SFTPGo development version for now, they will be included in v2.2.x.

Please try `ghcr.io/drakkan/sftpgo:edge`, `edge-slim`, `edge-distroless-slim` variants. It should work there

**Comment by drakkan** [comment]
sftpgo v2.2.0 was just released. It works with the stable version now
